simulator/main.cpp

```diff
@@ -34,13 +35,13 @@ int main( int argc, const char* argv[])
         /* running simulation */
         if ( !config::functional_only)
         {
-            PerfMIPS p_mips( config::disassembly_on);
+            PerfSim<MIPS> p_mips(bool(config::disassembly_on));
```

Comment:
Explicit conversion should be excessive here, was it a requirement from compiler?